src/comments.js

```diff
@@ -8,15 +8,8 @@ const {
   hasNewline,
   hasNewlineInRange,
 } = require("prettier").util;
-const {
-  concat,
-  join,
-  indent,
-  hardline,
-  cursor,
-  lineSuffix,
-  breakParent,
-} = require("prettier").doc.builders;
+const { concat, join, indent, hardline, cursor, lineSuffix, breakParent } =
```

Comment:
As I remeber concat is deprecated, but it is not a problem for such update

Reply:
That's right - we'll tackle that in the context of #1922 in a follow-up PR :+1: